services/py-api/src/service/mail_service/utils.py

```diff
@@ -0,0 +1,42 @@
+def load_email_participant_html_template(participant_name, team_name, invite_link):
```

Comment:
```python
def load_email_participant_html_template(participant_name, team_name, invite_link) -> Result[str,ValueError):
```
or some other type of error. I think raising ValueErrors on missing required params is obsolete.